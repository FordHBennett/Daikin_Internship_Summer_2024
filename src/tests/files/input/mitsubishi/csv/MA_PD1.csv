Tag Name,Address,Data Type,Respect Data Type,Client Access,Scan Rate,Scaling,Raw Low,Raw High,Scaled Low,Scaled High,Scaled Data Type,Clamp Low,Clamp High,Eng Units,Description,Negate Value
"PHS_PLC_Heartbeat","R00001",Short,1,R/W,100,,,,,,,,,,"",
"PLC_PHS_Heartbeat","R00000",Short,1,R/W,100,,,,,,,,,,"",
"PostResult_Message","RSH00042.030",String,1,R/W,100,,,,,,,,,,"",
"PostResult_PHSOK","R00040",Short,1,R/W,100,,,,,,,,,,"",
"PostResult_ReturnCode","R00041",Short,1,R/W,100,,,,,,,,,,"",
"RequestPostResults","R00005",Short,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_MachineFinishTime","R00119",Word,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_MachineStartTime","R00101",Word,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_OpeartionFinishTime","R00113",Word,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_OpeartionStartTime","R00107",Word,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_WorkPieceArrivalTime","R00095",Word,1,R/W,100,,,,,,,,,,"",
"6PointsManagement.6PM_WorkPieceReleaseTime","R00125",BCD,1,R/W,100,,,,,,,,,,"",
"Alarm&Call.PDError1","R00501",Short,1,R/W,100,,,,,,,,,,"",
"Alarm&Call.PDError2","R00502",Short,1,R/W,100,,,,,,,,,,"",
"Alarm&Call.PDError3","R00503",Short,1,R/W,100,,,,,,,,,,"",
"Alarm&Call.PDEstop","R00501.02",Short,1,R/W,100,,,,,,,,,,"",
"Alarm&Call.Unit_Failed","M0900",Boolean,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm1","R00501",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm10","R00510",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm11","R00511",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm12","R00512",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm13","R00513",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm14","R00514",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm15","R00515",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm16","R00516",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm17","R00517",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm18","R00518",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm19","R00519",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm2","R00502",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm20","R00520",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm21","R00521",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm22","R00522",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm23","R00523",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm24","R00524",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm25","R00525",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm26","R00526",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm27","R00527",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm28","R00528",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm29","R00529",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm3","R00503",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm30","R00530",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm31","R00531",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm32","R00532",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm33","R00533",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm34","R00534",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm35","R00535",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm36","R00536",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm37","R00537",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm38","R00538",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm39","R00539",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm4","R00504",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm40","R00540",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm41","R00541",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm42","R00542",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm43","R00543",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm44","R00544",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm45","R00545",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm46","R00546",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm47","R00547",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm48","R00548",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm49","R00549",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm5","R00505",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm50","R00550",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm51","R00551",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm52","R00552",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm53","R00553",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm54","R00554",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm55","R00555",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm56","R00556",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm57","R00557",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm58","R00558",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm59","R00559",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm6","R00506",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm60","R00560",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm61","R00561",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm62","R00562",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm63","R00563",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm64","R00564",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm65","R00565",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm66","R00566",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm67","R00567",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm68","R00568",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm69","R00569",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm7","R00507",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm70","R00570",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm71","R00571",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm72","R00572",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm73","R00573",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm74","R00574",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm75","R00575",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm76","R00576",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm77","R00577",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm78","R00578",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm79","R00579",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm8","R00508",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm80","R00580",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm81","R00581",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm82","R00582",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm83","R00583",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm84","R00584",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm85","R00585",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm86","R00586",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm87","R00587",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm88","R00588",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm89","R00589",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm9","R00509",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm90","R00590",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm91","R00591",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm92","R00592",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm93","R00593",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm94","R00594",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm95","R00595",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm96","R00596",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm97","R00597",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm98","R00598",BCD,1,R/W,100,,,,,,,,,,"",
"EquipmentAlarm.EquipmentAlarm99","R00599",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_AmbientMoisture","R00205",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_AmbientTemp","R00204",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_BalancedVacuumPres","R00208",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_CycleTime","R00203",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_ExhaustProcCompPres","R00217",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_ExhaustProcCompPres1","R00218",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_ExhaustProcCycleTime","R00219",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_HighPresTestChargingCycleTime","R00216",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_HighPresTestChargingPres","R00214",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_HighPresTestDecayPres","R00210",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_HPSOpsCheckResult","ZR000223",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_InitialUnitPres","R00211",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_InitialUnitPres1","R00212",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_LowPresTestChargingCycleTime","R00215",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_LowPresTestChargingPres","R00213",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_LowPresTestDecayPres","R00209",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_NGCode","RSH00201.004",String,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_PresDir","ZR000220",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_PresSpec","ZR000221",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_ProcessingResult","RSH00200.002",String,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_ReachedVacuumPressure","R00206",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_RegulatedPres","R00226",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_RegulatedPres1","R00227",Short,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_RetryCount","ZR000222",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_TrendDataFN","ZR000224.02",BCD,1,R/W,100,,,,,,,,,,"",
"TestResult.TestResult_VacuumProcessCycleTime","R00207",Short,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_CallSign","RSH00057.004",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_EQPTID","R00090",BCD,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_LineCode","RSH00087.006",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_ModelName","RSH00059.010",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_OperatorCode","RSH00091.008",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_PalletNo","RSH00083.008",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_Revision","RSH00069.002",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_SerialNo","RSH00078.010",String,1,R/W,100,,,,,,,,,,"",
"UnitInfo.UnitInfo_WO","RSH00070.016",String,1,R/W,100,,,,,,,,,,"",
