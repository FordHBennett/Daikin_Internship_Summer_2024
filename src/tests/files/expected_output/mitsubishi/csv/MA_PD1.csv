tag_name,address
TREND_DATA_FN,R<Int16>224.02
HIGH_PRES_TEST_DECAY_PRES,R<Int16>210
CYCLE_TIME,R<Int16>203
RETRY_COUNT,R<Int16>222
REVISION,R<String2>69
PALLET_NO,R<String8>83
INITIAL_UNIT_PRES,R<Int16>211
REGULATED_PRES,R<Int16>226
SN,R<String10>78
PROCESSING_RESULT,R<String2>200
EXHAUST_PROC_COMP_PRES1,R<Int16>218
REGULATED_PRES1,R<Int16>227
PRES_SPEC,R<Int16>221
VACUUM_PROC_CYCLE_TIME,R<Int16>207
LOW_PRES_TEST_CHARGING_CYCLE_TIME,R<Int16>215
REACHED_VACUUM_PRES,R<Int16>206
returnCode,R<Int16>41
HIGH_PRES_TEST_CHARGING_CYCLE_TIME,R<Int16>216
NG_CODE,R<String4>201
LINE_NAME,R<String6>87
LOW_PRES_TEST_DECAY_PRES,R<Int16>209
MODEL_NAME,R<String10>59
HPSOPS_CHECK_RESULT,R<Int16>223
AMBIENT_MOISTURE,R<Int16>205
LOW_PRES_TEST_CHARGING_PRES,R<Int16>213
EXHAUST_PROC_CYCLE_TIME,R<Int16>219
AMBIENT_TEMP,R<Int16>204
INITIAL_UNIT_PRES1,R<Int16>212
HIGH_PRES_TEST_CHARGING_PRES,R<Int16>214
PHS_PLC_Heartbeat,R<Int16>1
BALANCED_VACCUUM_PRES,R<Int16>208
EXHAUST_PROC_COMP_PRES,R<Int16>217
PHS_OK,R<Int16>40
PRES_DIR,R<Int16>220
REQUEST_TO_PHS,R<Int16>5
